internal/common/mqtt/mqttconnection.go

```diff
@@ -150,3 +150,23 @@ func (client *Client) Publish(message Message, topic string) error {
 	time.Sleep(time.Second)
 	return nil
 }
+
+var messageHandler MQTT.MessageHandler = func(client MQTT.Client, msg MQTT.Message) {
+
+	log.Info(logPrefix, "Topic ", msg.Topic(), " registered.. ", string(msg.Payload()), " is the payload")
+}
+
+//Subscribe is used to subscribe to a topic
+func (client *Client) Subscribe(topic string) error {
+	log.Info(logPrefix, "Subscribing to ", logmgr.SanitizeUserInput(topic))
```

Comment:
What about adding `// lgtm [go/log-injection]` in this line? (the same as the review from @t25kim in https://github.com/lf-edge/edge-home-orchestration-go/pull/506#issuecomment-1060073354)

Reply:
Done!!